### PR TITLE
Feature/153 home screen UI

### PR DESCRIPTION
**Pull request by Rebal67**

### CODE REVIEW

File: src/containers/MainContainer.tsx [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+import TCard from '../components/TCard';
+import useUserStore from '../store/userStore';
+
+export default function MainContainer() {
+    const user = useUserStore((state) => state.user);
+
+    return (
+        <View style={styles.container}>
+            <View style={styles.header}>
+                <TH2>{user.storage.username.username || 'username'}</TH2>
```
Comment (theblockstalk) [inline src/containers/MainContainer.tsx:15]: if username does not exist, better we show an error than a default

File: src/containers/MainContainer.tsx [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+                <Qricon height="200" width="100%" style={styles.marginTop} />
+                <TButtonContained style={[styles.button, styles.marginTop]} icon="qrcode-scan">
+                    {' '}
```
Comment (theblockstalk) [inline src/containers/MainContainer.tsx:18]: `{' '}` what is this for?

File: src/containers/MainContainer.tsx [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+                    Scan Qr Code
+                </TButtonContained>
+            </View>
+            <View style={styles.marginTop}>
+                <TP size={2}>Upcoming features</TP>
+                <ScrollView horizontal={true}>
+                    <TCard style={styles.card}>
+                        <TCard.Cover source={{ uri: 'https://source.unsplash.com/random/' }} />
```
Comment (theblockstalk) [inline src/containers/MainContainer.tsx:26]: use a value from config through settings

File: src/navigation/Drawer.tsx [hunk 4]
```diff
@@ -1,11 +1,12 @@
 import React from 'react';
 import { createDrawerNavigator, DrawerNavigationOptions } from '@react-navigation/drawer';
 
-import HomeScreen from '../screens/testScreen';
+import TestScreen from '../screens/testScreen';
```
Comment (theblockstalk) [inline src/navigation/Drawer.tsx:4]: can we remove this screen now?

File: package.json [hunk 5]
```diff
@@ -15,6 +15,7 @@
     "dependencies": {
       "@greymass/eosio": "^0.6.4",
       "@react-native-async-storage/async-storage": "~1.17.3",
+      "@react-navigation/bottom-tabs": "^6.5.0",
```
Comment (theblockstalk) [inline package.json:18]: what is this used for? I dont think we needed to implement bottom tabs

File: src/components/atoms/TBadge.tsx [hunk 6]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import { Badge, useTheme } from 'react-native-paper';
+
+type TBadgeProps = React.ComponentProps<typeof Badge>;
+export default function TBadge(props: TBadgeProps) {
+    const theme = useTheme();
+    const style = { backgroundColor: theme.colors.primary, color: 'white' };
```
[the remark this answers is not in the document]
Comment (Rebal67) [reply src/components/atoms/TBadge.tsx:7]: yes it doesn't use primary as backGroundColor it uses background as backGroundColor

File: src/components/atoms/THeadings.tsx [hunk 7]
```diff
@@ -14,7 +14,7 @@ export function TH1(props: THProps) {
 
 export function TH2(props: THProps) {
     return (
-        <Text {...props} style={styles.h2}>
+        <Text {...props} style={[styles.h2]}>
```
[the remark this answers is not in the document]
Comment (Rebal67) [reply src/components/atoms/THeadings.tsx:17]: i wanted to be able to add more than one style to it

File: src/containers/MainContainer.tsx [hunk 8]
```diff
@@ -0,0 +1,70 @@
...
+import useUserStore from '../store/userStore';
+
+export default function MainContainer() {
+    const user = useUserStore((state) => state.user);
+
+    return (
+        <View style={styles.container}>
+            <View style={styles.header}>
+                <TH2>{user.storage.username.username || 'username'}</TH2>
+                <Qricon height="200" width="100%" style={styles.marginTop} />
```
[the remark this answers is not in the document]
Comment (Rebal67) [reply src/containers/MainContainer.tsx]: done

File: src/containers/MainContainer.tsx [hunk 9]
```diff
@@ -0,0 +1,70 @@
...
+import { TButtonContained } from '../components/atoms/Tbutton';
+import { TH2, TP } from '../components/atoms/THeadings';
+import TCard from '../components/TCard';
+import useUserStore from '../store/userStore';
+
+export default function MainContainer() {
+    const user = useUserStore((state) => state.user);
+
+    return (
+        <View style={styles.container}>
```
[the remark this answers is not in the document]
Comment (Rebal67) [reply src/containers/MainContainer.tsx:13]: made a task

File: src/containers/MainContainer.tsx (hunk 2 above)
Comment (Rebal67) [reply src/containers/MainContainer.tsx:18]: auto added removed

File: src/containers/MainContainer.tsx (hunk 3 above)
Comment (Rebal67) [reply src/containers/MainContainer.tsx:26]: this will be commented out

File: src/navigation/Drawer.tsx (hunk 4 above)
Comment (Rebal67) [reply src/navigation/Drawer.tsx:4]: we will keep it for using

File: package.json (hunk 5 above)
Comment (Rebal67) [reply package.json]: ill comment the ui of the bottom navigation

File: src/containers/MainContainer.tsx (hunk 1 above)
Comment (Rebal67) [reply src/containers/MainContainer.tsx]: done

**Pull request merged**